util/dns_string_format.go

```diff
@@ -0,0 +1,41 @@
+package util
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+)
+
+func DNSHasNonStringIA5(c *x509.Certificate, isIssuer bool) (bool, error) {
+	oid := SANOID
```

Comment:
Don't we have a DNSNames field for both SAN and IAN extensions parsed out now? Why are we doing asn.1 marshalling here?